app/configuration/Config.scala

```diff
@@ -63,4 +64,9 @@ object Config {
   val sentryDsn = Try(new Dsn(config.getString("sentry.dsn")))
 
   lazy val Salesforce =  SalesforceConfig.from(config.getConfig("touchpoint.backend.environments").getConfig(stage), stage)
+
+  object GoCardless {
+    private val token = config.getString("gocardless.token")
+    val client = GoCardlessClient.create(token, Environment.SANDBOX)
```

Comment:
Fine for now to connect to sandbox by default, however it would be good to connect to the right environment here as we initialise a client that might be used later for other operations